lib/ansible/modules/cloud/digital_ocean/do_sshkeys.py

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/python
```

Comment:
I don't believe the current digital_ocean_sshkey module works today since DO shutdown the v1 route.